peps/pep-0756.rst

```diff
@@ -83,6 +83,7 @@ Add the following API to the limited C API version 3.14::
     int32_t PyUnicode_Export(
         PyObject *unicode,
         int32_t requested_formats,
+        uint32_t flags,
```

Comment:
I'm not sure it makes sense to make `flags` unsigned while `requested_formats` is signed.
Besides, it seems the two parameters could be combined? (it's highly unlikely we would grow more than 16 flags or 16 different formats)

Reply:
Ok, I reused *requested_formats* to pass flags.